src/routes.py

```diff
@@ -313,6 +325,12 @@ async def post_operation(
                 status_code=status.HTTP_403_FORBIDDEN,
                 detail=f"Entrypoint {entrypoint_name} is disabled.",
             )
+        except ConditionExceed:
+            logging.warning(f"A condition exceed the maximum defined.")
+            raise HTTPException(
+                status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                detail=f"A condition exceed the maximum defined.",
+            )
```

Comment:
I think it's a BAD_REQUEST as well. Also, we should probably start defining better errors so that the SDK can handle them too.

Reply:
> Also, we should probably start defining better errors so that the SDK can handle them too.
Totally agree !

Reply:
> I think it's a BAD_REQUEST as well.

Yeah I was hesitant but in my mind BAD_REQUEST is likely when you omit some parameters or parameters are invalid.
In this case, parameters are OK but you can't process the request because some conditions are not met to process it.
BUT it's a detail and I can change this :-)

Reply:
Oh you know I have no idea about those error codes anyway. Let's keep it that way.